homeassistant/components/shelly/strings.json

```diff
@@ -15,7 +16,7 @@
         }
       },
       "confirm_discovery": {
-        "description": "Do you want to set up the {model} at {host}?"
+        "description": "Do you want to set up the {model} at {host}?\n\nBefore set up, the battery-powered device must be woken up by pressing the button on the device."
```

Comment:
Do we need this for discovery? We should be able to gather all necessary info in the `async_step_discovery` such that `confirm_discovery` only requires user confirmation ?

Reply:
If the device is password protected, we cannot authorize when the device is asleep.